datasrc/network.py

```diff
@@ -403,11 +403,125 @@
 		NetStringStrict("m_Reason"),
 		NetBool("m_Force"),
 	]),
-	
+
 	### ModAPI
-	
+
 	NetMessage("ModAPI_Sv_Broadcast", [
 		NetStringStrict("m_pMessage"),
 	]),
-	
+
+
+	### NetGUI
+	## Server
+	NetMessage("ModAPI_Sv_GuiRemoveElement", [
+		NetIntAny("m_Type"),
+		NetIntAny("m_ID"),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiRequestData", [
+		NetIntAny("m_ID"),
+		NetIntAny("m_Type"),
+	]),
+
+	NetMessage("ModAPI_Sv_GuiUIRect", [
```

Comment:
Same as below, I just kept the names from vanilla. But I can change the name to `GuiRect` for you.